github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+|-------------------------------------------------------------------------------------------|
+| 6.3, 6.4, etc. (define specifically once we get a build number at end of release)         
+                                                                                            
+ 6.3 CU1, 6.4 CU1, etc. (define specifically once we a get build number at end of release)  
+                                                                                            
+ > Example 6.3.xxx.xx                                                                       
+ >                                                                                          
+ > All work items update automatically once the milestone is updated. (= minimal effort)    
+                                                                                            
+ There is not a vNext or backlog release.                                                   
+                                                                                            
+ > No Milestone = backlog                                                                   |
+
+Happy Path By issue
+===================
+
+Path below based on current State/Phase Labels
+
+****Question Happy Path****
+
+*Open -&gt; triaged -&gt; needs-more-info (optional) -&gt; in-progress -&gt; Closed. *
+
+****Bug or Documentation Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; need-more-info (optional) -&gt; understood (optional) -&gt; in -progress -&gt; Closed. *
+
+****Enhancement Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; design -&gt; in -progress -&gt; review -&gt; Closed. *
+
+> ****Questions, Defects and Doc issues****
+>
+> ***'Optional' Phases will apply to issues that are immediately understood allowing them to bypass many of the phases, moving the bug right to 'in-progress'.***
+
+****Enhancement Issues****
+
+> ***Where at 'review' the feature is reviewed for release and obtains any special sign-off needed from feature teams (if applicable). If not done (completed or has issues), go back to investigate (if needed) or move to design or in-progress for additional work. Repeat as needed. Once the feature can pass the review process (Whatever that may be for the given feature) it moves to closed***
+
+Issue Lifecycle
+===============
+
+Issues are reported with no labels, and no assignees. Issues then typically follow this cycle:
+
+1.  When an issue is reported it is sent to an appropriate area using the area label
+
+2.  If the type of issue is obvious, a type label is set
+
+3.  If the issue is not obvious, an investigating status label is set, and assignee is set
+
+4.  The individual assigned works on the issue, updating the issue, status and type labels
+
+If the assignee has found the issue does not belong to their area, they initiate a transfer by
+
+1.  Removing the area label
```

Comment:
I don't think it's necessary to drill out a process to "initiate a transfer".  I think it's implicit and natural with the principle of an issue being described by the 4 variables, for a developer to change the area when they realize it actually belongs elsewhere.  **Also**, an issue my touch multiple areas.